tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -47,6 +47,39 @@ public function testCodeWithoutReturnTypes($expected, $input = null)
         $this->doTest($expected, $input);
     }
 
+    /**
+     * @dataProvider provideCodeWithExtendsTypesCases
+     *
+     * @param mixed      $expected
+     * @param null|mixed $input
+     */
+    public function testCodeWithExtendsTypes($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideCodeWithExtendsTypesCases()
```

Comment:
can you please raise the PR to add it to 2.12 line?
i don't see how this test is related to feature you are adding, so let's push it to as lowest branch as possible

Reply:
This was probably a mistake that I did, I rebased the branch and removed unrelated code, please discard it then.